### PR TITLE
LRSD-7576 Update LSV category field to allow multiple values

### PR DESCRIPTION
**Pull request by ryanschuhler**

### CODE REVIEW

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java [hunk 1]
```diff
@@ -591,8 +591,8 @@ private JSONObject _transformSearchResults(JSONObject resultsJSONObject) {
 	)
 	private String _jiraSecurityVulnerabilityFieldAffectedVersionsDetails;
 
-	@Value("${liferay.customer.jira.security.vulnerability.field.category}")
-	private String _jiraSecurityVulnerabilityFieldCategory;
+	@Value("${liferay.customer.jira.security.vulnerability.field.categories}")
```
Comment (kylembischof) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java:594]: does `application-default.properties` config file need to updated with this change as well?

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/I18n/Language.json [hunk 2]
```diff
@@ -197,6 +198,7 @@
 	"find-a-project": "Find a project",
 	"finish-activation": "Finish Activation",
 	"first-name": "First Name",
+	"fix-version": "Fix Version",
 	"fix-versions": "Fix Versions",
```
Comment (kylembischof) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/I18n/Language.json:202]: if going singular for most frontend labeling, can we clean up any of the plural naming we arent using anymore?

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java (hunk 1 above)
Comment (jennyhchen) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java:594]: yes, I kept it "category" tho to match Jira field name (not the backend logic.)

Comment (ryanschuhler) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java:594]: good catch, fixed

**Comment by ryanschuhler** [comment]
ci:test:sf

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/I18n/Language.json (hunk 2 above)
Comment (ryanschuhler) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/I18n/Language.json:202]: we still use the plural keys on the individual ticket page. If we want to change/remove them we can, but i think it makes sense in that context still

**Comment by ryanschuhler** [comment]
ci:test:sf

**Comment by ryanschuhler** [comment]
ci:forward:force

**Comment by ryanschuhler** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/156822

**Pull request closed without merging**